packages/upload-client/src/index.js

```diff
@@ -63,13 +67,16 @@ export async function uploadFile(conf, file, options = {}) {
  * has the capability to perform the action.
  *
  * The issuer needs the `store/add` and `upload/add` delegated capability.
- * @param {import('./types.js').FileLike[]} files File data.
+ * @param {Iterable<FileLike> & { sorted?: boolean }} files  Files that should be in the directory.
```

Comment:
Pass `sorted` as an option? It's a bit odd to me that this option is passed as a property on the input files.

Do you think we should use "order" instead of "sorted"? They are needed a particular order, whether they arrived at that order because we sorted them doesn't matter.

As a suggestion for an option name: `customOrder: true`?

Reply:
> Do you think we should use "order" instead of "sorted"? They are needed a particular order, whether they arrived at that order because we sorted them doesn't matter.

Useful distinction. Thank you. I updated to prefer this notion of 'order' where appropriate

> As a suggestion for an option name: customOrder: true?

👍